pyproject.toml

```diff
@@ -2,11 +2,70 @@
 requires = ["setuptools>=45", "wheel", "setuptools_scm>=8.1.0"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "PyMeasure"
+# version is added dynamically
+authors = [
+    {name = "PyMeasure Developers"}
+]
+description = "Scientific measurement library for instruments, experiments, and live-plotting"
+# readme is merged dynamically from README and changelog
```

Comment:
Move this line to the "dynamic" entry, please.

Reply:
I disagree. The line `dynamic = ...` is very clear and doesn't need elaboration.

This is line is meant for a developer looking for a `readme = ...` entry. Instead of them incorrectly thinking we had forgotten it, with this comment I want to point them to the dynamic entry instead.